extensions/adobe/experience/journey/orchestration/journeyStepEventActionExecutionFieldsMixin.Json

```diff
@@ -0,0 +1,228 @@
+{
```

Comment:
None of your fields are namespaced. Are they intended to be `xdm:` fields or `campaign:` fields?

Reply:
updated